server/src/main/scala/dao/LiveRepository.scala

```diff
@@ -37,10 +35,8 @@ trait LiveRepository
     with DatabaseProvider
     with Config
     with ModelSlickInterop {
-  private val runtime                               = new DefaultRuntime {}
-  implicit val dbExecutionContext: ExecutionContext = runtime.platform.executor.asEC
 
-  implicit def provideDB[R](dbio: DBIO[R]): IO[RepositoryException, R] =
+  private def provideDB[R](dbio: DBIO[R]): IO[RepositoryException, R] =
```

Comment:
I guess I'm ok with making the provideDB explicit, I guess it's stylistic. What's your reasoning?

Reply:
For example below in [line 56](https://github.com/rleibman/full-scala-stack/pull/4/files#diff-ae8bcc13f827b7483566545f92a9e9fcL56) the `map` operation of an `DBIO` would require an `ExecutionContext`. By converting it to an `IO` we use the `map` of ZIO. `map` these doesn't need an `ExecutionContext` anymore.

That makes in turn the `dbExecutionContext` and the `runtime` obsolete and solves the reported ambiguity problem.

It would be possible to trigger the implicit conversion in some other way. (Adding type annotations for intermediate steps). But I think an explicit conversion is shorter and simpler here.